{{cookiecutter.project_name}}/docs/02_notebooks/config_example.ipynb

```diff
@@ -11,7 +11,7 @@
     "# The configuration utils\n",
     "\n",
     "You decided to opt in to use appliedAI's [accsr](https://github.com/appliedAI-Initiative/accsr) package to manage\n",
-    "configuration for you project - great choice :). In this template, you see a minimal usage of it. The `config.py`"
+    "configuration for you project - great choice :) In this template, you see a minimal usage of the `config.py`"
```

Comment:
pls rename notebooks to 01_.. and 02_.. so theyre listed in order. Also the intro should be named `00_intro.md`